### DISCUSSION TITLE
use ConfuserEX with winforms Application using Clickonce deployment

### DISCUSSION THREAD

**Discussion by RMorkas** [mkaring/ConfuserEx#343] [General]
Hi, I want to use ConfyserEX while I am building my application to generate the new version using Clickonce deployment.

I tried VS extension but it gives this error 
![image](https://user-images.githubusercontent.com/13630627/115605325-635e4b80-a2b0-11eb-8492-fbe410627b3c.png)

**Comment by mkaring** [comment]
I can't actually help you with the VS Extension of ConfuserEx, because this one isn't provided by this fork of ConfuserEx. I do not know where this extension is coming from.
You have a better chance getting help on the [Marketplace site for this extension][1].

[1]: https://marketplace.visualstudio.com/items?itemName=AvinabMalla.ConfuserExforVisualStudio2017-18023
